string chesspiece_pick
string chesspiece_place
bool chesspiece_move
---
bool result